amlb/results.py

```diff
@@ -319,7 +319,7 @@ def evaluate(self, metric):
         if hasattr(self, metric):
             return getattr(self, metric)()
         # raise ValueError("Metric {metric} is not supported for {type}.".format(metric=metric, type=self.type))
-        log.warning("Metric %s is not supported for %s!", metric, type=self.type)
+        log.warning("Metric {} is not supported for {}!".format( metric, self.type))
```

Comment:
FYI, this was not a typo, just a common practice to use the log formatter (here default printf-style) to avoid unnecessary string interpolation (e.g. if log level is not enabled).
I agree that the benefit is close to 0 here, but I mainly do this for consistency in all log statements.

Reply:
Sorry for the misunderstanding, but I am talking about the type argument. Specifically:
>>> import logging
>>> log = logging.getLogger()
>>> metric = 'bacloss'
>>> log.warning("Metric %s is not supported for %s!", metric, type='example')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.6/logging/__init__.py", line 1320, in warning
    self._log(WARNING, msg, args, **kwargs)
**TypeError: _log() got an unexpected keyword argument 'type'**

Reply:
oh right, my bad! Didn't even notice.  Good catch, thx!